build.gradle

```diff
@@ -1,29 +1,27 @@
 plugins {
-    id "java-library"
-    id "scala"
-    id "maven-publish"
-    id 'fabric-loom' version '0.10.30'
-    id "com.matthewprenger.cursegradle" version "1.4.0"
+	id 'java-library'
+	id 'scala'
+	id 'maven-publish'
+	id 'fabric-loom' version '1.5-SNAPSHOT'
 }
 
-sourceCompatibility = JavaVersion.VERSION_1_8
-targetCompatibility = JavaVersion.VERSION_1_8
+sourceCompatibility = JavaVersion.VERSION_17
+targetCompatibility = JavaVersion.VERSION_17
```

Comment:
Is Java 17 needed, this should still target Java 8 ideally.

Reply:
oh yea i can change that